src/game.cpp

```diff
@@ -1393,6 +1393,10 @@ ReturnValue Game::internalAddItem(Cylinder* toCylinder, Item* item, int32_t inde
 
 ReturnValue Game::internalRemoveItem(Item* item, int32_t count /*= -1*/, bool test /*= false*/, uint32_t flags /*= 0*/)
 {
+	if (!item) {
```

Comment:
This check is not necessary, since internalRemoveItem never receives nullptr, it only receives pointers (or shoulded) and pointers do not work to validate if it is pointing to valid memory.

in any case an assert is much more appropriate.